proto/one/operations/sample/analyte_result.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "jsonTicksDateTime.proto";
+import "enum_import_status.proto";
+option csharp_namespace ="ONE.Models.CSharp";
+
+message AnalyteResult {
+
+    // AuthTwinRefId
+    google.protobuf.StringValue authTwinRefId = 1;
+    
+    // The Sample Number (letter and numbers) that describe the sample
+    string sampleNumber = 2;
+    
+    // The name of the analyte.  (The subject of the activity)
+    string analyteName = 3;
```

Comment:
Is analyteName unique enough to identify an analyte from?  Will the name in the file match what is in Rio?

Reply:
I'd have to look at more requests from the web app, but I could see potentially adding the AnalyteID.  The property bag looks like...
"propertyBag": "{\"authTwinRefId\":\"4b317892-d2d6-430e-93a9-802a48e75a26\",
\"analyteId\":\"f6443a4f-f7b8-4e85-883d-200fd7507851\",
\"result\":\"<8\",
\"methodDetectionLimit\":\"3\",
\"reportingLimit\":\"3\",

Reply:
I like AnalyteId, does the propertyBag even have the AnalyteName?

Reply:
I like the AnalyteId better. As Dan mentioned, the analyte name is not in the property bag. We would have to make a call to get the analyte to get the name.

Reply:
The users will only know the name, so we have to go with analyte name instead of id.